corehq/apps/cloudcare/templates/formplayer/session_list.html

```diff
@@ -42,12 +42,12 @@ <h1 class="page-title">{% trans "Incomplete Forms" %}</h1>
     </div>
   </td>
   <td class="module-column module-column-name module-column-name-session">
-    <h3>
+    <h3 id="<%- labelId %>">
       <b><%- title %></b>
       <% if (caseName) { %>(<%- caseName %>)<% } %>
       opened <%- humanDateOpened %>
     </h3>
-    <div class="module-delete-control">
+    <div class="module-delete-control" role="button" tabindex="0" aria-label="Delete form">
```

Comment:
I think this should be translated, with `trans_html_attr`

Reply:
Done.